gettsim/benefits/wohngeld_dag.py

```diff
@@ -270,3 +274,42 @@ def _wohngeld_eink_abzüge_ab_2016(
         abzüge = pd.Series(dtype=float)
 
     return abzüge
+
+
+def _wohngeld_eink(
+    tu_id,
+    haushalts_größe,
+    _wohngeld_eink_abzüge,
+    _wohngeld_abzüge,
+    _wohngeld_brutto_eink,
+    _wohngeld_sonstiges_eink,
+    wohngeld_params,
+):
+    _wohngeld_eink_abzüge_per_tu = _wohngeld_eink_abzüge.groupby(tu_id).transform("sum")
+
+    vorläufiges_eink = (1 - _wohngeld_abzüge) * (
+        _wohngeld_brutto_eink + _wohngeld_sonstiges_eink - _wohngeld_eink_abzüge_per_tu
+    )
+
+    unteres_eink = [
+        wohngeld_params["min_eink"][hh_size]
+        if hh_size < 12
+        else wohngeld_params["min_eink"][12]
+        for hh_size in haushalts_größe
+    ]
+
+    return vorläufiges_eink.clip(lower=unteres_eink)
+
+
+def haushalts_größe(hh_id):
```

Comment:
```suggestion
def haushaltsgröße(hh_id):
```
No reason to change spelling of words.

Reply:
will do.